sources/Tools/DatasetHandler.py

```diff
@@ -113,10 +118,17 @@ def __init__(self, parse: bool = False, train: bool = False):
         Public methods
     """
 
-    def save_header(self, header_file: str = default_header_file):
+    def save_header(self, header_file: str = default_header_files["advanced"]):
         self._save_npy(header_file, self.analysed_header)
 
-    def load_header(self, header_file: str = default_header_file):
+    def load_header(self):
```

Comment:
why did you delete the header_file parameter?

Reply:
Cause the arg wasn't used anywhere in the code by the called function and the new way to access to it is more efficient, it depend on the self.args content.

Reply:
it s a public tool, it should be very generic, the only reason it's not a staticmethod it s because _load_npy is call, please change it